scripts/create-kubestellar-demo-env.sh

```diff
@@ -150,3 +189,4 @@ echo "Checking the new clusters are in the OCM inventory and label them"
 kubectl --context its1 get managedclusters
 kubectl --context its1 label managedcluster cluster1 location-group=edge name=cluster1
 kubectl --context its1 label managedcluster cluster2 location-group=edge name=cluster2
+
```

Comment:
Maybe a "done" message is appropriate here?

Reply:
Roger that - included now